ktor-core/test/org/jetbrains/ktor/tests/http/ContentTypeTest.kt

```diff
@@ -89,4 +89,10 @@ class ContentTypeTest {
             }
         }
     }
+
+    Test
+    fun `content type with empty parameters block`() {
+        ContentType.parse("text/plain; ")
```

Comment:
Added it. But the main goal here is to test that we are not crashing at such string as we did before